File: Beecrowd/1048 Aumento de Salário/main.py
Sal = float(input())
	
if Sal >= 0 and Sal <= 400:
    imposto = Sal * 0.15
    Sal = Sal + imposto
    print("Novo salario: %0.2f" %Sal)
    print("Reajuste ganho: %0.2f" %imposto)
    print("Em percentual: 15 %")

elif Sal <= 800:
    imposto = Sal * 0.12
    Sal = Sal + imposto
    print("Novo salario: %0.2f" %Sal)
    print("Reajuste ganho: %0.2f" %imposto)
    print("Em percentual: 12 %")

elif Sal <= 1200:
    imposto = Sal * 0.10
    Sal = Sal + imposto
    print("Novo salario: %0.2f" %Sal)
    print("Reajuste ganho: %0.2f" %imposto)
    print("Em percentual: 10 %")

elif Sal <= 2000:
    imposto = Sal * 0.07
    Sal = Sal + imposto
    print("Novo salario: %0.2f" %Sal)
    print("Reajuste ganho: %0.2f" %imposto)
    print("Em percentual: 7 %")

else:
    imposto = Sal * 0.04
    Sal = Sal + imposto
    print("Novo salario: %0.2f" %Sal)
    print("Reajuste ganho: %0.2f" %imposto)
    print("Em percentual: 4 %")

File: Beecrowd/1048 Aumento de Salário/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	double Sal, imposto;
	
	cin >> Sal;
	
	cout << fixed;
	cout.precision(2);
	if (Sal >= 0.0 && Sal <= 400.0) {
		imposto = Sal * 0.15;
		Sal = Sal + imposto;
		cout << "Novo salario: " << Sal << endl;
		cout << "Reajuste ganho: " << imposto << endl;
		cout << "Em percentual: 15 %" << endl;
	}
	else if (Sal <= 800.0) {
		imposto = Sal * 0.12;
		Sal = Sal + imposto;
		cout << "Novo salario: " << Sal << endl;
		cout << "Reajuste ganho: " << imposto << endl;
		cout << "Em percentual: 12 %" << endl;
	}
	else if (Sal <= 1200.0) {
		imposto = Sal * 0.10;
		Sal = Sal + imposto;
		cout << "Novo salario: " << Sal << endl;
		cout << "Reajuste ganho: " << imposto << endl;
		cout << "Em percentual: 10 %" << endl;
	}
	else if (Sal <= 2000.0) {
		imposto = Sal * 0.07;
		Sal = Sal + imposto;
		cout << "Novo salario: " << Sal << endl;
		cout << "Reajuste ganho: " << imposto << endl;
		cout << "Em percentual: 7 %" << endl;
	}
	else {
		imposto = Sal * 0.04;
		Sal = Sal + imposto;
		cout << "Novo salario: " << Sal << endl;
		cout << "Reajuste ganho: " << imposto << endl;
		cout << "Em percentual: 4 %" << endl;
	}
 
    return 0;
}